Products/Zuul/catalog/model_catalog_tool.py

```diff
@@ -110,7 +110,7 @@ def _build_query(self, types=(), paths=(), depth=None, query=None, filterPermiss
 
         @return: tuple (AdvancedQuery query, not indexed filters dict)
         """
-        available_indexes = self.model_catalog_client.get_indexes()
+        indexed, not_indexed = self.model_catalog_client.get_indexes()
```

Comment:
`not_indexed` variable name is misleading. `stored` or something similar would be better. We will have fields in `not_indexed` that are in `indexed`. ie objectImplements is indexed and stored.